README.md

```diff
@@ -41,3 +50,83 @@ After configuring the service proxies, you can look toward the top, under User S
 
 You will be able to add the name of the key you added (`my_new_proxy` in this example) so that the users can select it.
 The entry with a star is going to be the default selection if a user does not configure it. You can delete all other entries from here to force a single one.
+
+## Modifying Browser Settings
+
+You can configure the following browser settings in URLDownloader:
+
+- User agent(`user_agent`): This is the value of `User-Agent` in request headers.
+- Window size(`window_size`): The screen resolution of the browser.
+- Request headers (`headers`/`request_headers`): Request headers used to fetch URLs.
+
+Administrators can set the default values for these settings from the URLDownloader configuration page in the **Service Variables** section.
+These settings will be used by every single submissions sent to URLDownloader.
+
+Here is an example of `default_browser_settings [json]` service variable:
+
+```
+# iPhone 11 Pro configuration:
+{
+    "window_size": "375x812",
+    "user_agent": "Mozilla/5.0 (iPhone; CPU iPhone OS 14_6 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.3 Mobile/15E148 Safari/604.1",
+    "request_headers":
+    {
+        "Sec-CH-UA-Mobile": "?1"
+    }
+}
+```
+
+Users can also modify these settings per URL submission by submitting an Assemblyline URI file.
+Here is an example:
+
+```
+# Assemblyline URI file 1
+uri: https://sample_webpage.com/
+
+browser_settings:
```

Comment:
What happens if a user specifies anything under browser_settings.request_headers? Will it get rejected, merged with headers, or one will overwrite the other?